### PR TITLE
Upgrade to .NET 5

### PR DESCRIPTION
**Pull request by Mailaender**
This eliminates the `writable-sdk` hack.

### CODE REVIEW

File: net.openra.OpenRA.yaml [hunk 1]
```diff
@@ -188,20 +156,18 @@ modules:
         dest: LocalNugetPackages
         sha256: 8ed8fe846d8b1523ee56829b783e0408f0e5a2814fe30a2647cf5149ed578d1c
       - type: file
-        # Source: https://download.ip2location.com/lite/IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP
-        path: IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP
-      - type: script
-        commands:
-          - "# NOP"
-        dest-filename: fetch-geoip.sh
+        url: https://github.com/OpenRA/GeoIP-Database/releases/download/monthly/IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP
```
Comment (Unrud) [inline net.openra.OpenRA.yaml:159]: The URL is not stable. The build will fail when the file is updated.

File: net.openra.OpenRA.yaml [hunk 2]
```diff
@@ -188,20 +156,18 @@ modules:
...
         path: openra.metainfo.xml.in
     buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/dotnet5/bin
     build-commands:
       - if git rev-parse 2>/dev/null; then make version; fi
-      - make TARGETPLATFORM=unix-generic
-      - make prefix=/app install
-      - make prefix=/app install-linux-shortcuts
+      - make RUNTIME=net5 TARGETPLATFORM=unix-generic all
```
Comment (Unrud) [inline net.openra.OpenRA.yaml]: The `RUNTIME` variable doesn't yet exist in this release.

**Comment by Unrud** [comment]
I suggest delaying the switch to .Net until the next playtest release.

File: net.openra.OpenRA.yaml (hunk 2 above)
Comment (Mailaender) [reply net.openra.OpenRA.yaml]: Hmm, yes seems that I am only using the `dotnet msbuild` but the runtime is still Mono.

File: net.openra.OpenRA.yaml (hunk 1 above)
Comment (Mailaender) [reply net.openra.OpenRA.yaml:159]: The URL is stable. Only the file/checksums change. There is a reason this file is updated regularly.

**Comment by Unrud** [comment]
See #20

**Pull request closed without merging**